facts/secrets/openwrt-example.yaml

```diff
@@ -21,7 +22,7 @@ wired:
   networks:
     # interface names cant have special characters
     - name: 'mgmt'
-      ifname: 'eth1.103 eth1.503'
+      ifname: 'eth1'
```

Comment:
I changed this in the example so when Im building the openwrt imgs I dont have to setup a vlan locally. I usually forget to do that and then spend a bit figuring out why I cant connect